connollyp/WIT-SWE-MEAL-PREP
# user login database connection

## Overview

When a user goes to login, the database is checked. If User exists and password is correct they can login. Otherwise they cannot.

## Setup

You need to connect a database.

In the mealprep folder create a file called '.env'

Put the following in that file (replace with correct info, see discord for aws database info)

DATABASE_HOST='host'
DATABASE_USER='username'
DATABASE_PASSWORD='password'

The .env file hold environment variables we don't want exposed to the public

To create a user, connect to the database using a mySQL client of choice

and run an insert sql script like the following

`INSERT INTO User (Username, Password, First_Name, Last_Name)
VALUES ("Jakewd18", "password1", "Jake", "Dickinson");`

You can also use my user if you don't want to do that.

## Test steps

Make sure you are in the mealprep folder.

run `npm start` in one terminal tab
and
run `node server.js` in another

server.js is the connection from the database to the web app

got to localhost:3000 in web browser

log in with username 'Jakewd18' and password 'password1'

They are case sensitive at the moment.
